convert.py

```diff
@@ -24,21 +25,34 @@ def convert(flags):
         if content:
             processed = process_content(i, content)
             block = generate_schema_block(processed)
-            blocks.append(block)
+
+            try:
```

Comment:
Think the body of this `try` should move into it's own function now as this is getting pretty long